### DISCUSSION TITLE
`Submit Review` not visible if the PR is short

### DISCUSSION THREAD

**Discussion by 1ucian0** [community/community#49185] [Repositories]
### Select Topic Area

Bug

### Body

When reviewing a particularly small PR, it is not possible to click on the `Submit Review`:
<img width="1125"  src="https://user-images.githubusercontent.com/766693/222923225-799b9749-eb69-4484-8054-92b824d120dd.png">

Scrolling does not work, not even in the review box.

A workaround is to extend the lines in the file, if possible.

**Comment by casperson** [comment]
Yep, I've seen this same issue multiple times. One-liners are even worse.

![Screenshot 2023-05-19 at 12 51 51 PM](https://github.com/community/community/assets/6778814/f5edfaf8-a566-45b5-a591-e77662421f1f)

In this case there were more lines in the file so I could expand it to be able to approve, but the review box should always show entirely, regardless of whether or not the PR is short or long.

![Screenshot 2023-05-19 at 12 52 22 PM](https://github.com/community/community/assets/6778814/8014bc7f-e431-4b15-9ee6-556240f72060)

**Comment by DNin01** [comment]
Pressing <kbd>ctrl</kbd><kbd>enter</kbd> typically submits a comment, so maybe that will work?

Resizing the comment box to make it smaller could also help.

If the radio buttons are also offscreen, pressing <kbd>tab</kbd> enough times will set focus on them, then you can use <kbd>↓</kbd> to choose an option. And if you press <kbd>tab</kbd> again, the `Submit review` button will be focused.

Or you can simply zoom out.

Nevertheless, this isn't ideal for the user experience.
